src/features/styles/theme/breakpoints.scss

```diff
@@ -0,0 +1,17 @@
+$breakpoints: (
```

Comment:
Do we need all these 9 breakpoints? I remember we already had more than this number and that was so confusing but I think 9 breakpoints are still a lot.
In most CSS frameworks such as MUI and TailwindCSS there are about 5 breakpoints.

Also the naming `wide` and `large` is not clear to show which one is bigger.
The order for `tablet` is different in compare to `phone` and `desktop` and `large` is smaller than `wide`.